pkg/license/download.go

```diff
@@ -72,7 +73,7 @@ func (do *DownloaderOptions) Validate() error {
 		}
 		// And no cache dir was specified
 		if do.CacheDir == "" {
-			dir, err := ioutil.TempDir(os.TempDir(), "license-cache-")
+			dir, err := ioutil.TempDir("", "license-cache-")
```

Comment:
Should we use https://golang.org/pkg/os/#MkdirTemp?

Reply:
Yes, fixed as suggested :+1: